_includes/new-page.md

```diff
@@ -35,7 +35,7 @@ The first paragraph above has an example of a markdown link to a different site.
 For links within USDS, there is a special URL format that is needed to make things work across development, staging and on the live site. The brackets-followed-by-parentheses still applies. Inside those parentheses, the first part of the URL must be double curly braces containing `site.baseurl`
 
 - Yes: [this link]({{ site.baseurl }}/mission) will always be right.
```

Comment:
I need to fix this documentation. [this link](mission) is the correct approach with this change.